prov/rxm/src/rxm_ep.c

```diff
@@ -444,6 +444,22 @@ static void rxm_ep_txrx_pool_destroy(struct rxm_ep *rxm_ep)
 	free(rxm_ep->buf_pools);
 }
 
+static int rxm_multi_recv_pool_init(struct rxm_ep *rxm_ep)
+{
+	struct ofi_bufpool_attr attr = {
+		.size		= sizeof(struct rxm_recv_entry),
+		.alignment	= 16,
+		.max_cnt	= 0,
+		.chunk_cnt	= 16,
+		.alloc_fn	= NULL,
+		.init_fn	= NULL,
+		.context	= rxm_ep,
+		.flags		= OFI_BUFPOOL_NO_TRACK | OFI_BUFPOOL_HUGEPAGES,
```

Comment:
Huge pages are a limited resource.  We don't want to request them unless absolutely needed.  And they're not needed for this structure.